datanode/generated/proto/rpc/peer_streaming.proto

```diff
@@ -0,0 +1,117 @@
+//  Copyright (c) 2017-2018 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package rpc;
+
+message KafkaOffset {
+    int64 checkPointOffset = 1; // archiving/backfill/snapshot checkpoint kafka offset
+    int64 commitOffset = 2; // ingestion committed kafka offset
+}
+
+message BackfillCheckpoint {
+    int64 redoFileID = 1; // checkpoint redolog file id, dimension table only
+    uint32 redoFileOffset = 2; // checkpoint redolog file offset, dimension table only
+}
+
+message ArchiveVersion {
+    uint32 archiveVersion = 1; // archive version, fact table only
+    uint32 backfillSeq = 2; // backfillSeq, fact table only
```

Comment:
what is this..?

Reply:
each backfill will increment a sequence number, which is part of the archive batch's version